tests/behat/uploadFileToMoodle.feature

```diff
@@ -43,3 +43,31 @@ Feature: upload the resource in oCIS to moodle
     And I click on "//*[@class='fp-filename-field']/p[text()='testfile.txt']" "xpath_element"
     And I click on "Select this file" "button"
     Then I should see "testfile.txt"
+
+  Scenario Outline: click refresh button to get latest resource on Personal/Project drive
+    Given "admin" has created the project space "ProjectMoodle"
+    And I should not see "ProjectMoodle"
+    When I refresh the file-picker
+    Then I should see "ProjectMoodle"
+    And I click on "<drive>" "link"
+    And user "admin" has uploaded a file inside space "<drive>" with content "some content" to "testfile.txt"
+    And I should not see "testfile.txt"
+    When I refresh the file-picker
+    Then I should see "testfile.txt"
+    Examples:
+      | drive         |
+      | ProjectMoodle |
+      | Personal      |
```

Comment:
hmm after you click on ProjectMoodle drive how could you see testfile.txt if you upload resource on Personal drive?

Reply:
`uploaded a file inside space "<drive>"`

The file is uploaded to the matching drive for each example.